APMrover2/mode_steering.cpp

```diff
@@ -12,8 +12,8 @@ void ModeSteering::update()
         return;
     }
 
-    float desired_steering, desired_speed;
-    get_pilot_desired_steering_and_speed(desired_steering, desired_speed);
+    float desired_steering, desired_speed, desired_lateral;
+    get_pilot_desired_steering_and_speed(desired_steering, desired_speed, desired_lateral);
```

Comment:
If we used a separate get_pilot_desired_lateral() then we could leave steering unchanged.  At the moment it's a bit odd because we decode the lateral but never use it.  As a minimum we should add a comment that the lataral is not currently used.

Reply:
Thanks for the feedback. I added lateral_out in steering and acro only because they use get_pilot_desired_steering_and_throttle() which also has lateral_out because it uses get_pilot_input(). I'm actually still not sure how lateral control should behave in these two modes. I think I will add a new method to get lateral input so this way I can remove lateral_out from functions that don't use lateral at the moment,